x/leverage/types/errors.go

```diff
@@ -38,4 +38,6 @@ var (
 		1126,
 		"market total collateral would exceed MaxCollateralShare",
 	)
+	ErrNotUToken = sdkerrors.Register(ModuleName, 1127, "denom is not a uToken")
+	ErrUToken    = sdkerrors.Register(ModuleName, 1128, "denom is a uToken")
```

Comment:
Let's be a bit more direct here:
```suggestion
	ErrNotUToken = sdkerrors.Register(ModuleName, 1127, "denom should be a uToken")
	ErrUToken    = sdkerrors.Register(ModuleName, 1128, "denom should not be a uToken")
```